cmd/zapi/cmd/post/post.go

```diff
@@ -116,6 +116,19 @@ func abspaths(paths []string) ([]string, error) {
 	var err error
 	out := make([]string, len(paths))
 	for i, path := range paths {
+		// Special handling for paths starting with S3:
```

Comment:
How about calling `uri, err := ParseURI` (instead of the prefix check), and then setting `out[i] = uri.String()` of the result? That would pair well with using the ParseURI in zqd. For file paths, ParseURI will send the absolute path, so the filepath.Abs call below could be removed.

Reply:
Good suggestion, I will make that change